docs/technical-documentation/Security_Assessment.md

```diff
@@ -0,0 +1,95 @@
+# Security Assessment Policy-Hub 
+
+|                           |                                                                                                |
+| ------------------------- | ---------------------------------------------------------------------------------------------- |
+| Contact for product       | [@evegufy](https://github.com/evegufy) <br> [@jjeroch](https://github.com/jjeroch)             |
+| Security responsible      | [Szymon Kowalczyk](szymon.kowalczyk@zf.com) |
+| Version number of product |                                                                                           |
+| Dates of assessment       | 2024-02-16: Assessment                                                                      |
+| Status of assessment      | ASSESSMENT DRAFT                                                                            |
+
+
+## Product Description
+
+To be updated
+
+### Important Links
+
+To be updated
+
+## Data Flow Diagram
+
+```mermaid
+flowchart LR
+
+    CU-Own(Company user)
+    NC1("Potential new company (admin)")
+    CU-Shared1(Company user)
+
+    K("Keycloak (REST API)")
+
+    PH(Policy Hub API)
+    PHD[("Postgres Database \n \n (Data created with \nApplication deployment)")]
+
+    subgraph operator[Operator IdP]
+      subgraph centralidp[centralidp Keycloak]
+      K
+      end
+
+    subgraph sharedidp[shared IdP Keycloak]
+        subgraph companyrealm1[Company realm]
```

Comment:
rename to companyrealm[sharedIdP Keycloak or ownIdP]

Reply:
Done, although I have named it literally "sharedIdP Keycloak or ownIdP" Please let me know if that's what you had in mind for this change.